lesson01/assignment/scripts.js

```diff
@@ -30,4 +31,105 @@ document. addEventListener("DOMContentLoaded", function(){
     inputField.value = ''
   })
 
+  //Map Form
+  // Find the map form, and handle its submit event
+  const mapForm = document.querySelector('#map-form')
+
+  mapForm.addEventListener('submit', function (event) {
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array that we can transform based on user input
+    let map_data = [1, 2, 3, 4, 5, 6]
```

Comment:
If the `p` tag I commented on above had a selector, it could be populated here with something like:
```
labelText.innerHTML = `Array: [${map_data.toString(', ')}]`
```